_i18n/de/coc.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "QMK Verhaltensregeln"
+layout: qmk
+lang: "de"
+---
+
+# Verhaltensregeln (Code of Conduct)
+
+QMK strebt danach eine inklusive, tolerante, und einladende Community zu sein. Wir ermutigen Beteiligung von jedem, unabhängig vom Alter, Behinderung, ethnischer Zugehörigkeit, Geschlechtsidentität und -Ausdruck, Erfahrungsniveau, Nationalität, Aussehen, politischer Ansichten, Abstammung, Religion, oder sexuellen Identität oder Ausrichtung.
```

Comment:
```suggestion
QMK strebt danach eine inklusive, tolerante, und einladende Community zu sein. Wir ermutigen die Beteiligung von jedem Individuum, unabhängig vom Alter, Behinderung, ethnischer Zugehörigkeit, Geschlechtsidentität und -Ausdruck, Erfahrungsniveau, Nationalität, persönlichem Auftreten, politischer Ansicht, Abstammung, Religion, oder sexueller Identität oder Ausrichtung.
```

Reply:
Maybe use *jedem/er* or the lesser known *jedeR* to refer to both female and male.
```suggestion
QMK strebt danach eine inklusive, tolerante, und einladende Community zu sein. Wir ermutigen Beteiligung von jedem/er, unabhängig vom Alter, Behinderung, ethnischer Zugehörigkeit, Geschlechtsidentität oder -ausdruck, Erfahrungsniveau, Nationalität, persönlichem Auftreten, politischer Ansicht, Abstammung, Religion, oder sexueller Identität oder Ausrichtung.
```

Reply:
I chose the neutrum genus because some people identify with neither male nor female. Those might not relate to 'Jedem/er'. I know that this might be virtue signalling by a long shot but it is ment for a group of people who are likely to need the CoC more than I do ;)